native/dgeqrf.go

```diff
@@ -0,0 +1,93 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"github.com/gonum/blas"
+	"github.com/gonum/lapack"
+)
+
+// Dgeqrf computes the QR factorization of the m×n matrix a using a blocked
+// algorithm. Please see the documentation for Dgeqr2 for a description of the
+// parameters at entry and exit.
+//
+// work is vector of temporary storage with length at least n, and will panic
```

Comment:
An example of `Work is ...` comment. At least Dgeqrf should be inserted before will, but also see above.